specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2021-11-01-preview/managedClusters.json

```diff
@@ -3752,6 +3755,10 @@
           "type": "boolean",
           "description": "(DEPRECATING) Whether to enable Kubernetes pod security policy (preview). This feature is set for removal on October 15th, 2020. Learn more at aka.ms/aks/azpodpolicy."
         },
+        "enableNamespaceResources": {
+          "type": "boolean",
+          "description": "Enable namespace as Azure resource."
```

Comment:
It'd be good to see more of a description than this. I think we should be looking to answer the following questions:

- Do we have an article talking about namespace as a resource and how to use it? If so can we link it here? If not an article, is there at least a REST API reference document for the resource in question that we can point customers to, so they at least know what APIs setting this to true enables them to call?
- Are there requirements on when this can be set? Must it be set only on cluster create? Can it be enabled later? Can it be disabled after it's enabled? If it's disabled after it's enabled what happens?
- What's the default if this parameter is not specified?

Reply:
You can use the `title`/`description` split if you get a lot and want to split it into a more "summary" doc like you have here and then a "details" doc (which would go into `description`, the summary into `title`).

Reply:
- On article, here's the documentation [link](https://aka.ms/NamespaceARMResource) we have created. Will update the target for this aka.ms link to point to the new documentation page for this feature which will be written up when we are closer to the feature release date. @shashankbarsin has the action item on this.
- On when this can be set, the property can be enabled/disabled on cluster create and update. If it is disabled after enabling, user will not be able to see the namespace ARM resources.
-  Default value is false.

Reply:
> On when this can be set, the property can be enabled/disabled on cluster create and update. If it is disabled after enabling, user will not be able to see the namespace ARM resources.

> Default value is false.

Can you put those into the documentation, so that the customer also knows the answers to these questions?

Reply:
@matthchr , updated the title and description